package.json

```diff
@@ -2,29 +2,38 @@
   "name": "web",
   "version": "1.0.0",
   "description": "angular my-uw",
-  "main": "index.js",
   "scripts": {
-    "test": "cd uw-frame-components; bower install; cd ..; karma start uw-frame-components/karma.conf.js  --single-run; cd uw-frame-components; rm -rf bower_components",
-    "build-all": "npm run build-java; npm run build-static",
-    "build-docs" : "cd docs; ./build.sh;",
-    "build-java": "cd uw-frame-java; mvn clean package",
-    "build-static": "cd uw-frame-static; ./build.sh",
-    "jetty": "cd uw-frame-java; mvn jetty:run",
-    "static": "cd uw-frame-static/target; http-server",
-    "docs" : "cd docs/target; http-server",
-    "clean": "rm -rf uw-frame-components/bower_components; cd uw-frame-java; mvn clean; cd ..; rm -rf uw-frame-static/target"
+    "pretest": "cd uw-frame-components && bower --config.interactive=false install",
```

Comment:
Oh yeah, I meant to call this out - it's a naming convention that if you run script `foo` the runner always looks for a `prefoo` to run beforehand and a `postfoo` to run afterward, which gives you a rudimentary dependency scheme.  Since they'll each run in their own thread, we get to drop the `cd ..` bits